src/core/public/styles/chrome/_banner.scss

```diff
@@ -21,3 +21,22 @@
     top: $kbnHeaderBannerHeight + $euiHeaderHeightCompensation;
   }
 }
+
+// overriding padding on the body element added by EUI
+.kbnBody.kbnBody--hasHeaderBanner.kbnBody--projectLayout.euiBody--headerIsFixed {
+  padding-top: $kbnHeaderBannerHeight + $euiHeaderHeightCompensation;
+
+  // overriding `top` positioning of the project side nav, and flyouts
+  // overriding `top` positioning of the project app menu toolbar
+  &.euiBody--headerIsFixed .euiCollapsibleNav,
+  &.euiBody--headerIsFixed:not(.euiDataGrid__restrictBody) .euiFlyout,
+  .header__actionMenu {
+    top: $kbnHeaderBannerHeight + $euiHeaderHeightCompensation;
+  }
+
+  // overriding `height` calculation of the project side nav, and flyouts
+  &.euiBody--headerIsFixed .euiCollapsibleNav
```

Comment:
It still is getting cut off at the bottom. By chance, ought there be a comma here? This calculation looks correct, but I don't believe it ever gets applied, as is.
```suggestion
  &.euiBody--headerIsFixed .euiCollapsibleNav,
```

Reply:
D'oh! Sorry about that - I think it was a bit of hasty code cleanup. Fixed in d57714c5ca3

<img width="1604" alt="image" src="https://github.com/elastic/kibana/assets/908371/61e797a5-133c-4b6d-a709-51686b9af54e">